### ISSUE TITLE
Background image in desktop mode

### ISSUE THREAD

**Issue by almejo** [tim-sueberkrueb/ubuntu-hangups#37]
I am using hangups in desktop mode. I was trying to change the background but it says "sorry, there aren't any apps installed that can provide this type of content'

Is it posible to fix this in desktop mode using a file chooser dialog?

**Comment by timsueberkrueb** [comment]
@almejo yes, I actually hope that - at least when the platforms converge - this dialog will be functional on the desktop as well (when phone apps are installable on the desktop/the content hub is fully integrated). 
I'll see what I can do regarding the file chooser. The only difficulty is to detect whether the app is running on the desktop or the phone.
For now you can just copy the image file to `ubuntu-hangups/custom_chat_background.png`
and adjust the relevant line in `settings.json`:

``` json
"custom_chat_background": "custom_chat_background.png",
```

**Comment by almejo** [comment]
Great!!!!

So, this is a know issue? should i close it and wait?

On Tue, Jan 5, 2016 at 11:18 AM, Tim Süberkrüb notifications@github.com
wrote:

> @almejo https://github.com/almejo yes, I actually hope that - at least
> when the platforms converge - this dialog will be functional on the desktop
> as well (when phone apps are installable on the desktop/the content hub is
> fully integrated).
> I'll see what I can do regarding the file chooser. The only difficulty is
> to detect whether the app is running on the desktop or the phone.
> For now you can just copy the image file to
> ubuntu-hangups/custom_chat_background.png
> and adjust the relevant line in settings.json:
> 
> "custom_chat_background": "custom_chat_background.png",
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/tim-sueberkrueb/ubuntu-hangups/issues/37#issuecomment-169012649
> .

## 

Alejandro Vera
http://www.recicleta.cl

**Comment by timsueberkrueb** [comment]
@almejo Yes, it's normal currently, but nevertheless a problem, leave it open :)
